portfolio/src/main/webapp/js/pets.js

```diff
@@ -0,0 +1,12 @@
+function slideShow() {
+    setTimeout(slideShow, 1000);
+    const img = document.querySelectorAll(".image");
+    for(var i = 0; i < img.length; i++) {
+        img[i].style.display = "none";
+    }
+
+    const imgIndex = Math.floor(Math.random() * img.length);
```

Comment:
just so I understand, we're picking a random image here to show as the first pic?

Reply:
Yes

Reply:
cool, do you mind adding a comment noting that?